templates/living_documentation_regime/_us_index_root_level_page_template.md

```diff
@@ -0,0 +1,9 @@
+---
+title: User Stories
+toolbar_title: User Stories
+description_title: User Stories
+description: >
+  The comprehensive list of AqueDuct Project's User Stories.
+date: {date}
+weight: 0
+---
```

Comment:
I get we add to it, but should we maybe borrow from Jinja and other templating engines where they replace named parts like `{% content %}` with content at specific spots? Like so you don't know where it will be.

Reply:
Actually why are we not using Jinja ?

Reply:
Great idea - created a new issue to add jinja2 - https://github.com/AbsaOSS/living-doc-generator/issues/113.